packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -205,7 +208,7 @@ export default function LabeledField(props: Props) {
             <React.Fragment>
                 <BodyText
                     style={[
-                        styles.textWordBreak,
+                        styles.helperTextMessage,
```

Comment:
Simplifying some styling! I removed `textWordBreak` and included it in the styles that needed it (`styles.helperTextMessage` and `styles.label`)